main.go

```diff
@@ -11,8 +15,16 @@ const (
 )
 
 func main() {
+	log_filename := os.Args[1]
+
+	if _, err := os.Stat(log_filename); errors.Is(err, os.ErrNotExist) {
+		message := "File " + log_filename + " does not exist."
+		fmt.Println(message)
+		return
```

Comment:
Exit with a non-zero exit code on failure.
```suggestion
		os.Exit(2)
```